views/index.ejs

```diff
@@ -8,10 +8,14 @@
     <h1>Shopping for Cookies</h1>
 
 
-    <strong>Items in cart: <!-- Print the number of items in the cart here --></strong>
+    <strong>Items in cart: <%= cart.length %></strong>
     <ul>
       <!-- List the names of the items in your cart here -->
-
+        <% cart.forEach(function (cartItem){ %>
```

Comment:
remove the space between function and (cartItem)

`<% cart.forEach(function(cartItem){ %>`